api/client/webclient/webconfig.go

```diff
@@ -126,7 +126,11 @@ type FeatureLimits struct {
 // UIConfig provides config options for the web UI served by the proxy service.
 type UIConfig struct {
 	// ScrollbackLines is the max number of lines the UI terminal can display in its history
-	ScrollbackLines int `json:"scrollbackLines,omitempty"` //nolint:unused // marshaled in config/configuration.go for WebConfig
+	ScrollbackLines int `json:"scrollbackLines,omitempty"`
+	// DisableIncludeRequestableRequests will turn off the option in the web UI to request resources with the "includeRequestable" flag.
+	// This will make resource requests NOT include the "RequiresRequest" field and thus not display both types of resources
+	// to users ("have access" and "requires request") in the unified resources UI.
+	DisableIncludeRequestableRequests bool `json:"disableIncludeRequestableRequests,omitempty"`
```

Comment:
Having a bool field that you set to true to disable is counter-intuitive and often causes confusion.

WDYT about instead having a string/enum instead?

```
show_resources: accessible # or 'all', defaults to accessible if unset
```

Reply:
Whats funny is I originally had this as "Allow" with default to true but thought "no, having to configure an "allow" to false is counter intuitive". Seems like its confusing all around!

I think `show_resources` might be a bit too ambiguous but can probably be cleared up with a comment. It needs to state that if this is set to `accessible` than you won't even be able to select "all" in the dropdown. 

I'll fiddle with it and resort to show_resources if I don't think of a better option. 

Thank you!

Reply:
Updated to `show_resources` with the valid values `accessable_only` and `requestable` (default)